src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+
+    public String getRandomColor() {
+        return Color.values()[random.nextInt(Color.values().length)].name();
```

Comment:
you need to call `toLowerCase()` as well

`name()` will return you `WHITE` instead of `white`
```suggestion
        return Color.values()[random.nextInt(Color.values().length)].name().toLowerCase();
```